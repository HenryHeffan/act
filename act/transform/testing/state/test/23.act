defproc  bar (bool q, r, s)
{
  prs {
    q & r -> s-
  }
}

defproc fuz (bool a, b)
{
  prs {
    a -> b-
  }
  bar z;
  bar zz;
}

defproc foo (bool x)
{ 
  fuz a[4];
}

foo f;
