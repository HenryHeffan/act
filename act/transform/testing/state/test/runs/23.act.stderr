In expanding foo<> (23.act:17)
In expanding foo<> (23.act:17)
In expanding fuz<> (inst: a) (23.act:8)
WARNING: Process `foo<>': local variable `a[0].a': no driver
WARNING: Process `foo<>': local variable `a[1].a': no driver
WARNING: Process `foo<>': local variable `a[2].a': no driver
WARNING: Process `foo<>': local variable `a[3].a': no driver
WARNING: Process `fuz<>': local variable `z.q': no driver
WARNING: Process `fuz<>': local variable `z.r': no driver
WARNING: Process `fuz<>': local variable `zz.q': no driver
WARNING: Process `fuz<>': local variable `zz.r': no driver
